cmd/crio/main.go

```diff
@@ -588,6 +588,11 @@ func main() {
 			logrus.Fatal(err)
 		}
 
+		// Immediately upon start up, write our new version file
+		if err := version.WriteVersionFile(server.CrioVersionPath, gitCommit); err != nil {
+			logrus.Errorf(err.Error())
```

Comment:
```suggestion
			logrus.Error(err)
```

Do we want to stop there or continue the execution?

Reply:
Hm, I opted to have it be fatal. I'm not married to that though @mrunalp WDYT